src/index.js

```diff
@@ -1,19 +1,58 @@
-import _ from 'lodash';
-import printMe from './print.js';
+// import _ from 'lodash';
+import './style.css';
 
-function component() {
-  const element = document.createElement('div');
-  const btn = document.createElement('button');
+const mainListCard = document.querySelector('.main-list-card');
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const todoTask = [
+  {
+    description: 'Typing',
+    isCompleted: false,
+    index: 3,
+  },
+  {
+    description: 'Listening',
+    isCompleted: false,
+    index: 2,
+  },
+  {
+    description: 'Take Nap',
+    isCompleted: false,
+    index: 1,
+  },
+  {
+    description: 'Gaming',
+    isCompleted: false,
+    index: 0,
+  },
+];
```

Comment:
- [OPTINAL] On page load, the tasks should appear in order of the `index` values for each task. I strongly suggest you set the property index to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5). HINT thought_balloon you can modify indexes of your code and make sure the first index is 1 and not zero  this way when you create a todo list, the index starts from 1 instead of 0